fluent-plugin-events/lib/fluent/plugin/in_events.rb

```diff
@@ -43,90 +44,139 @@ def configure(conf)
       def start
         super
         initialize_resource_version
-        start_watcher_thread
-        start_configmap_flush_thread
+        start_monitor
       end
   
-      def close
-        @watcher.each &:finish
+      def stop
+        log.debug "Clean up before stopping completely"
```

Comment:
can we be sure to document in the README how to enable debug logs for a fluentd plugin (is there a way to only enable debug logs for a specific plugin?) so that the customer/support team can enable these debug logs as needed?

Reply:
Yes, in the README Sam put in troubleshooting steps that includes turning on debug logs for metrics. We'll make that generic to all three data types afterwards

Reply:
👍  Thanks

Reply:
Thanks, I've just updated the plugin with steps to enable debug logs for a specific plugin